plugins/SitesManager/lang/en.json

```diff
@@ -88,6 +88,20 @@
         "SiteWithoutDataMessageDisappears": "This message will disappear as soon as some data is tracked for this website.",
         "SiteWithoutDataIgnoreMessage": "Don't show this message for the next hour",
         "SiteWithoutDataChoosePreferredWay": "Choose your preferred way of getting set up",
+        "SiteWithoutDataCloudflare": "Cloudflare",
+        "SiteWithoutDataCloudflareDescription": "You can use the Matomo App in Cloudflare to track data, follow the instructions from this %1$sguide%2$s.",
+        "SiteWithoutDataCloudflareIntro": "If you use Cloudflare, you can start tracking data in Matomo seamlessly using the Matomo app available on Cloudflare.",
+        "SiteWithoutDataCloudflareFollowStepsIntro": "Follow this steps to set it up:",
```

Comment:
@Stan-vw This change looks good to you ?

Reply:
'steps' is a plural noun, therefore requires a [plural demonstrative](https://www.britannica.com/dictionary/eb/qa/This-these-that-and-those), 'this step' vs 'these steps'.

Reply:
Small tweaks:
        "SiteWithoutDataCloudflare": "Cloudflare",
        "SiteWithoutDataCloudflareDescription": "You can use the Matomo App in Cloudflare to track data, follow the instructions from %1$sthis guide%2$s.",
        "SiteWithoutDataCloudflareIntro": "If you use Cloudflare, you can start tracking data in Matomo seamlessly using the Matomo App available in Cloudflare.",
        "SiteWithoutDataCloudflareFollowStepsIntro": "Follow these steps to set it up:",

Reply:
whoops, you gotta teach me how to make the text so beautiful 😅

Reply:
@Stan-vw If you find the lines to tweak on the 'Files Changed' tab up top, then click the blue + by the line number (or drag for multiple lines) it will add a comment against those lines, in the comment box that appears, on the toolbar, there is a page icon with a plus/minus symbol which will add a copy of the lines as a suggestion in the comment edit box, you can then edit this copy of the lines to what you think they should be. 

This way they get formatted nicely, can be seen as a 'diff' on the files changes list, can be committed in a single click with attribution and it also creates a comment on the main conversation page here :slightly_smiling_face: